public/components/event_analytics/explorer/events_views/surrounding_flyout.tsx

```diff
@@ -6,15 +6,16 @@
 import './docView.scss';
 import React, { useEffect, useState, Fragment } from 'react';
 import {
-  EuiButton,
-  EuiButtonEmpty,
+  EuiSmallButton,
+  EuiSmallButtonEmpty,
   EuiButtonIcon,
   EuiCallOut,
-  EuiFieldNumber,
+  EuiCompressedFieldNumber,
   EuiFlexGroup,
   EuiFlexItem,
   EuiFlyoutBody,
   EuiFlyoutHeader,
+  EuiCompressedFormRow,
```

Comment:
Might have missed a form row here.

Reply:
No. There is a `EuiFormRow` that has its own `display`.